### PR TITLE
Polishing the design of the project view

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
- fixed the layout of the ProjectView
image

### CODE REVIEW

**Comment by kgevorkyan** [comment]
@akuleshov7 I can't see the window with standard test suites selection after your latests PR,
may be you could also fixed it here?

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt [hunk 1]
```diff
@@ -428,48 +435,54 @@ class ProjectView : RComponent<ProjectExecutionRouteProps, ProjectViewState>() {
...
-                                            attrs.defaultValue = text
-                                            attrs.disabled = true
-                                            attrs {
-                                                onChangeFunction = {
-                                                    val tg = it.target as HTMLInputElement
-                                                    val newValue = tg.value
-                                                    newProjectInformation[header] = newValue
+                            projectInformation
+                                .forEach { (header, text) ->
+                                   // div("control-group form-inline") {
```
Comment (kgevorkyan) [inline save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt:461]: dead code?

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt [hunk 2]
```diff
@@ -428,48 +435,54 @@ class ProjectView : RComponent<ProjectExecutionRouteProps, ProjectViewState>() {
...
+                                                            val tg = it.target as HTMLInputElement
+                                                            val newValue = tg.value
+                                                            newProjectInformation[header] = newValue
+                                                        }
+                                                    }
                                                 }
                                             }
                                         }
-                                    }
+                                   // }
```
Comment (kgevorkyan) [inline save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt:483]: same

**Comment by akuleshov7** [comment]
> Could you also please provide the screenshot of how it looks like now?
> 
> Major changes I already saw online, but we also talked about movements of `save`, `cancel` buttons, etc

attached

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt (hunk 2 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt]: fixed. thanks

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt (hunk 1 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt]: fixed. thanks

**Comment by akuleshov7** [comment]
image

**Review by kgevorkyan — APPROVED** [review]
Did you try to make save/cancel buttons colorized (green, red)?
Will it be more ugly, that's why you use default color?

**Comment by akuleshov7** [comment]
> Did you try to make save/cancel buttons colorized (green, red)? Will it be more ugly, that's why you use default color?

exactly

**Pull request merged**